test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -26,20 +26,24 @@ class TestFunctional(common_utils.TorchaudioTestCase):
 
     backend = "default"
 
-    def assert_batch_consistency(self, functional, batch, *args, atol=1e-8, rtol=1e-5, seed=42, **kwargs):
-        n = batch.size(0)
-
+    def assert_batch_consistency(self, functional, *, inputs, args=None, kwargs=None, atol=1e-8, rtol=1e-5, seed=42):
```

Comment:
I am having a second thought about this. Maybe using `partial` is better. What do you think?

Reply:
`partial` may make the `functional` not consistent. If the function doesn't have `kwargs`, then it's a pure `F.xyz` method, otherwise it's a `partial`. I feel like `kwargs` as dictionary is fine.

Reply:
> `partial` may make the `functional` not consistent. 

That should not matter because the test is interested in whether the returned value are consistent. If the function passed belongs to `torchaudio.functional` has no effect on the test logic.

Reply:
That's true. The test logic won't be affected in either case. The advantage of `kwargs` is that the test methods can follow the same pattern.

Reply:
[partial](https://docs.python.org/3/library/functools.html) allows args and kwargs, so it's essentially just moving the argument binding outside of the helper method.
The concern / responsibility of the helper method is the function to be tested and Tensor inputs, so delegating bidding of the other arguments to call site is less error-prone.